uitools/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml

```diff
@@ -216,6 +213,20 @@ Control {
                 }
             }
         }
+
+        Button {
+            anchors {
+                top: fieldsLayout.bottom
+                topMargin: 20
+                horizontalCenter: parent.horizontalCenter
+            }
+            Layout.alignment: Qt.AlignHCenter
```

Comment:
:x: If you are removing the button from the Layout I don't think this line is required.